>> anaconda-3-5.3.1 has been loaded.
>> python-3.9.7 has been loaded.
>> cuda-10.1_cudnn-7.6.5 has been loaded.
WARNING: Skipping page https://pypi.org/simple/pip/ because the GET request got Content-Type: .The only supported Content-Type is text/html
WARNING: Skipping page https://pypi.org/simple/pip/ because the GET request got Content-Type: .The only supported Content-Type is text/html
Traceback (most recent call last):
  File "main.py", line 1, in <module>
    from src.experimenter_uncommented import gaframework, gaframework_gpu
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/experimenter_uncommented.py", line 3, in <module>
    from src.abstracts.geneticalgo_integ import SNPGeneticAlgo
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/geneticalgo_integ.py", line 5, in <module>
    from .fitness import *
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/fitness.py", line 3, in <module>
    from src.abstracts.cpu_timer import *
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/cpu_timer.py", line 26, in <module>
    conf_save(filename, timer_params)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/cpu_timer.py", line 6, in conf_save
    conf_save(filename, timer_params)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/cpu_timer.py", line 6, in conf_save
    conf_save(filename, timer_params)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/cpu_timer.py", line 6, in conf_save
    conf_save(filename, timer_params)
  [Previous line repeated 949 more times]
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/cpu_timer.py", line 5, in conf_save
    doc = yaml.safe_dump(timer_params, out)
  File "/home/arlan.uy/.local/lib/python3.7/site-packages/yaml/__init__.py", line 306, in safe_dump
    return dump_all([data], stream, Dumper=SafeDumper, **kwds)
  File "/home/arlan.uy/.local/lib/python3.7/site-packages/yaml/__init__.py", line 278, in dump_all
    dumper.represent(data)
  File "/home/arlan.uy/.local/lib/python3.7/site-packages/yaml/representer.py", line 28, in represent
    self.serialize(node)
  File "/home/arlan.uy/.local/lib/python3.7/site-packages/yaml/serializer.py", line 54, in serialize
    self.serialize_node(node, None, None)
  File "/home/arlan.uy/.local/lib/python3.7/site-packages/yaml/serializer.py", line 108, in serialize_node
    self.serialize_node(value, node, key)
  File "/home/arlan.uy/.local/lib/python3.7/site-packages/yaml/serializer.py", line 109, in serialize_node
    self.emit(MappingEndEvent())
  File "/home/arlan.uy/.local/lib/python3.7/site-packages/yaml/emitter.py", line 115, in emit
    self.state()
  File "/home/arlan.uy/.local/lib/python3.7/site-packages/yaml/emitter.py", line 393, in expect_first_block_mapping_key
    return self.expect_block_mapping_key(first=True)
  File "/home/arlan.uy/.local/lib/python3.7/site-packages/yaml/emitter.py", line 403, in expect_block_mapping_key
    self.expect_node(mapping=True, simple_key=True)
  File "/home/arlan.uy/.local/lib/python3.7/site-packages/yaml/emitter.py", line 244, in expect_node
    self.expect_scalar()
  File "/home/arlan.uy/.local/lib/python3.7/site-packages/yaml/emitter.py", line 268, in expect_scalar
    self.process_scalar()
  File "/home/arlan.uy/.local/lib/python3.7/site-packages/yaml/emitter.py", line 533, in process_scalar
    self.write_plain(self.analysis.scalar, split)
  File "/home/arlan.uy/.local/lib/python3.7/site-packages/yaml/emitter.py", line 1132, in write_plain
    self.stream.write(data)
  File "/opt/hpcc/CPU/anaconda/3-5.3.1/lib/python3.7/codecs.py", line 328, in reset
    IncrementalDecoder.reset(self)
RecursionError: maximum recursion depth exceeded
srun: error: saliksik-gpu-05: task 2: Exited with exit code 1
Traceback (most recent call last):
  File "main.py", line 303, in <module>
    main()
  File "main.py", line 301, in main
    program_main(sys.argv)
  File "main.py", line 188, in program_main
    gaframework(system, test_cases_path, savefile_name)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/experimenter_uncommented.py", line 26, in gaframework
    execute_experiment(rssnp, ga, gaeval, ga_params, loadfile_name, start_new, start_from_gen)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/experimenter_uncommented.py", line 49, in execute_experiment
    gaeval.run(ga, rssnp, stats_run['population_size'], stats_run['fitness_function'],  gaeval.no_of_run, gaeval.no_of_gen, stats_run['mutation_rate'], loadfile_name, stats_run['selection_func'], start_new, start_from_gen)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/GAevaluator_integ.py", line 24, in run
    self.no_of_run = self.ga_params['run_total'] + self.ga_params['runs_pending']
TypeError: 'NoneType' object is not subscriptable
srun: error: saliksik-gpu-05: task 3: Exited with exit code 1
Traceback (most recent call last):
  File "main.py", line 303, in <module>
    main()
  File "main.py", line 301, in main
    program_main(sys.argv)
  File "main.py", line 188, in program_main
    gaframework(system, test_cases_path, savefile_name)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/experimenter_uncommented.py", line 26, in gaframework
    execute_experiment(rssnp, ga, gaeval, ga_params, loadfile_name, start_new, start_from_gen)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/experimenter_uncommented.py", line 49, in execute_experiment
    gaeval.run(ga, rssnp, stats_run['population_size'], stats_run['fitness_function'],  gaeval.no_of_run, gaeval.no_of_gen, stats_run['mutation_rate'], loadfile_name, stats_run['selection_func'], start_new, start_from_gen)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/GAevaluator_integ.py", line 38, in run
    run_fitness = genetic_algo.simulate(system, size, function, generations, mutation_rate, path_name, run, selection_func, start_from_gen)            
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/geneticalgo_integ.py", line 273, in simulate
    timer_write_run(run_index)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/cpu_timer.py", line 40, in timer_write_run
    timer_params['run_indexes'][run_index_state] = {}
TypeError: string indices must be integers
Traceback (most recent call last):
  File "main.py", line 303, in <module>
    main()
  File "main.py", line 301, in main
    program_main(sys.argv)
  File "main.py", line 188, in program_main
    gaframework(system, test_cases_path, savefile_name)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/experimenter_uncommented.py", line 26, in gaframework
    execute_experiment(rssnp, ga, gaeval, ga_params, loadfile_name, start_new, start_from_gen)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/experimenter_uncommented.py", line 49, in execute_experiment
    gaeval.run(ga, rssnp, stats_run['population_size'], stats_run['fitness_function'],  gaeval.no_of_run, gaeval.no_of_gen, stats_run['mutation_rate'], loadfile_name, stats_run['selection_func'], start_new, start_from_gen)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/GAevaluator_integ.py", line 38, in run
    run_fitness = genetic_algo.simulate(system, size, function, generations, mutation_rate, path_name, run, selection_func, start_from_gen)            
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/geneticalgo_integ.py", line 273, in simulate
    timer_write_run(run_index)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/cpu_timer.py", line 40, in timer_write_run
    timer_params['run_indexes'][run_index_state] = {}
TypeError: string indices must be integers
srun: error: saliksik-gpu-05: tasks 0-1: Exited with exit code 1
Traceback (most recent call last):
  File "main.py", line 303, in <module>
    main()
  File "main.py", line 301, in main
    program_main(sys.argv)
  File "main.py", line 188, in program_main
    gaframework(system, test_cases_path, savefile_name)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/experimenter_uncommented.py", line 26, in gaframework
    execute_experiment(rssnp, ga, gaeval, ga_params, loadfile_name, start_new, start_from_gen)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/experimenter_uncommented.py", line 49, in execute_experiment
    gaeval.run(ga, rssnp, stats_run['population_size'], stats_run['fitness_function'],  gaeval.no_of_run, gaeval.no_of_gen, stats_run['mutation_rate'], loadfile_name, stats_run['selection_func'], start_new, start_from_gen)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/GAevaluator_integ.py", line 38, in run
    run_fitness = genetic_algo.simulate(system, size, function, generations, mutation_rate, path_name, run, selection_func, start_from_gen)            
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/geneticalgo_integ.py", line 273, in simulate
    timer_write_run(run_index)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/cpu_timer.py", line 40, in timer_write_run
    timer_params['run_indexes'][run_index_state] = {}
TypeError: string indices must be integers
Traceback (most recent call last):
  File "main.py", line 303, in <module>
    main()
  File "main.py", line 301, in main
    program_main(sys.argv)
  File "main.py", line 188, in program_main
    gaframework(system, test_cases_path, savefile_name)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/experimenter_uncommented.py", line 26, in gaframework
    execute_experiment(rssnp, ga, gaeval, ga_params, loadfile_name, start_new, start_from_gen)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/experimenter_uncommented.py", line 49, in execute_experiment
    gaeval.run(ga, rssnp, stats_run['population_size'], stats_run['fitness_function'],  gaeval.no_of_run, gaeval.no_of_gen, stats_run['mutation_rate'], loadfile_name, stats_run['selection_func'], start_new, start_from_gen)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/GAevaluator_integ.py", line 38, in run
    run_fitness = genetic_algo.simulate(system, size, function, generations, mutation_rate, path_name, run, selection_func, start_from_gen)            
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/geneticalgo_integ.py", line 273, in simulate
    timer_write_run(run_index)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/cpu_timer.py", line 40, in timer_write_run
    timer_params['run_indexes'][run_index_state] = {}
TypeError: string indices must be integers
Traceback (most recent call last):
  File "main.py", line 303, in <module>
    main()
  File "main.py", line 301, in main
    program_main(sys.argv)
  File "main.py", line 188, in program_main
    gaframework(system, test_cases_path, savefile_name)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/experimenter_uncommented.py", line 26, in gaframework
    execute_experiment(rssnp, ga, gaeval, ga_params, loadfile_name, start_new, start_from_gen)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/experimenter_uncommented.py", line 49, in execute_experiment
    gaeval.run(ga, rssnp, stats_run['population_size'], stats_run['fitness_function'],  gaeval.no_of_run, gaeval.no_of_gen, stats_run['mutation_rate'], loadfile_name, stats_run['selection_func'], start_new, start_from_gen)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/GAevaluator_integ.py", line 38, in run
    run_fitness = genetic_algo.simulate(system, size, function, generations, mutation_rate, path_name, run, selection_func, start_from_gen)            
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/geneticalgo_integ.py", line 273, in simulate
    timer_write_run(run_index)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/cpu_timer.py", line 40, in timer_write_run
    timer_params['run_indexes'][run_index_state] = {}
TypeError: string indices must be integers
Traceback (most recent call last):
  File "main.py", line 303, in <module>
    main()
  File "main.py", line 301, in main
    program_main(sys.argv)
  File "main.py", line 188, in program_main
    gaframework(system, test_cases_path, savefile_name)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/experimenter_uncommented.py", line 26, in gaframework
    execute_experiment(rssnp, ga, gaeval, ga_params, loadfile_name, start_new, start_from_gen)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/experimenter_uncommented.py", line 49, in execute_experiment
    gaeval.run(ga, rssnp, stats_run['population_size'], stats_run['fitness_function'],  gaeval.no_of_run, gaeval.no_of_gen, stats_run['mutation_rate'], loadfile_name, stats_run['selection_func'], start_new, start_from_gen)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/GAevaluator_integ.py", line 38, in run
    run_fitness = genetic_algo.simulate(system, size, function, generations, mutation_rate, path_name, run, selection_func, start_from_gen)            
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/geneticalgo_integ.py", line 273, in simulate
    timer_write_run(run_index)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/cpu_timer.py", line 40, in timer_write_run
    timer_params['run_indexes'][run_index_state] = {}
TypeError: string indices must be integers
srun: error: saliksik-gpu-05: tasks 4,6-7: Exited with exit code 1
srun: error: saliksik-gpu-05: task 5: Exited with exit code 1
